Code/Client/Private/NetImgui_CmdPackets.h

```diff
@@ -112,6 +119,11 @@ struct alignas(8) CmdInput
 	bool							mCompressionUse		= false;	// Server would like client to compress the communication data
 	bool							mCompressionSkip	= false;	// Server forcing next client's frame data to be uncompressed
 	uint8_t							PADDING[4]			= {};
+
+#if IMGUI_VERSION_NUM >= 18700
```

Comment:
Hmm. Do we need back compability if imgui introduces this breaking changes?)

Reply:
I haven't looked in detail about the input changes Dear ImGui introduced, but there are a lot of people pre Dear ImGui 1.87. Game projects tend to minimize changes until the game is shipped, so plenty of people will still be relying on the older input system. On my side, I would like the NetImgui Server application to continue being able to communicate with multiple clients using different Dear ImGui versions simultaneously.

Reply:
As for how to support it properly, I guess we could copy the new ImGui input enum into NetImgui and only enable it when the version is 1.87, so the enum is always valid. Instead of using windows virtual keys override, we now can use this same enum instead. This is just a quick idea, without having research the problem yet.